### DISCUSSION TITLE
Unable to access source and donate.

### DISCUSSION THREAD

**Discussion by Pointy-hair** [Foblex/f-flow#13] [Q&A] [answered]
Unable to access source and donate.

**Comment by siarheihuzarevich** [comment, marked as answer]
We plan to open access to the source code no earlier than next year. Thank you for your interest. But you can use the library for free, documentation is available at [Documentation](https://foblex.com/flow/documentation/get-started)

**Comment by Pointy-hair** [comment]
Hi Thank you very much for your work.
The issue is I couldn't use the component without access to source as if there was any problems I wouldn't have the safety of fixing it. Sorry I will have to look elsewhere.

**Comment by Pointy-hair** [comment]
Welcome to the @foblex/flow
@foblex/flow is an Angular library designed to simplify the creation and manipulation of dynamic flow. Provides components for flows, nodes, and connections, automating node manipulation and inter-node connections.

Features
Node and Connection creation and manipulation.
Connection and Node events.
Reassigning Nodes and Connections.
Customizable Node and Connection templates.
Drag-and-drop support.
Zoom and pan support.
Getting Started and Documentation
Visit the [Foblex Flow Documentation](https://foblex.com/flow/documentation/get-started) to learn how to install and use the library in your Angular project.

Support and Community
For questions, feedback, and support, visit the [Foblex Portal](https://www.foblex.com/flow/home) to connect with the community and the development team. You can also report [issues](https://github.com/Foblex/flow/issues) and request [features](https://github.com/Foblex/flow/discussions) on the [GitHub repository](https://github.com/Foblex/flow).

License and Access to Source Code
This library is available for use under the MIT License, allowing free use, modification, and distribution of the minified version of the library in your projects.

**If you are interested in accessing the full source code of the library for purposes of study, modification, or deeper integration with your projects, we offer separate licenses for the source code. Purchasing such a license will grant you the right to use, modify, and distribute the source code according to the terms set forth in the licensing agreement.**

For more information and to purchase a source code license, please contact us at [Foblex](https://www.foblex.com/flow/home).

**Comment by siarheihuzarevich** [comment]
I solved your problem. Source code is available in the repository
